$OpenBSD: patch-src_actions_cpp,v 1.1 2015/06/19 16:23:33 dcoppa Exp $

commit aa8c47d5782664b7d9786e1195f7950e29ee9a21
Author: Andrzej Rybczak <electricityispower@gmail.com>
Date:   Sun May 17 16:30:11 2015 +0200

actions: respect regular expression config when searching in text
fields

commit 20a4e486cc36e40afc202e3794772e402b1d3c0f
Author: Andrzej Rybczak <electricityispower@gmail.com>
Date:   Sat May 23 19:33:20 2015 +0200

lastfm: use sink argument and fix possible usage of m_service after
delete

--- src/actions.cpp.orig	Sat May  2 21:16:11 2015
+++ src/actions.cpp	Fri Jun 19 16:30:29 2015
@@ -1932,8 +1932,8 @@ void Find::run()
 	Statusbar::print("Searching...");
 	auto s = static_cast<Screen<NC::Scrollpad> *>(myScreen);
 	s->main().removeProperties();
-	if (findme.empty() || s->main().setProperties(NC::Format::Reverse, findme, NC::Format::NoReverse))
-		Statusbar::print("Done");
+  if (findme.empty() || s->main().setProperties(NC::Format::Reverse, findme, NC::Format::NoReverse, Config.regex_type))
+    Statusbar::print("Done");
 	else
 		Statusbar::print("No matching patterns found");
 	s->main().flush();
@@ -2314,7 +2314,7 @@ void ShowArtistInfo::run()
 	
 	if (!artist.empty())
 	{
-		myLastfm->queueJob(LastFm::ArtistInfo(artist, Config.lastfm_preferred_language));
+		myLastfm->queueJob(new LastFm::ArtistInfo(artist, Config.lastfm_preferred_language));
 		myLastfm->switchTo();
 	}
 #	endif // HAVE_CURL_CURL_H
